datafusion/contributor-guide/architecture.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
However, I still haven't quite gotten the subdirectories working yet

http://localhost:4000/datafusion/user-guide/introduction.html

Shows me a "not found" error (as Jekyll has created `datafusion/user-guide/introduction/index.html` (the directory) rather than `datafusion/user-guide/introduction.html` (the file)

I ran out of time now, but I will play with this a bit more

Reply:
Fixed